tests/test_filter_panels.py

```diff
@@ -0,0 +1,63 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+@pytest.mark.nondestructive
+def test_edit_filter_panels(base_url, selenium):
+    """Open Treeherder page, open Filters Panel, select one filter,
+    verify results"""
+    page = TreeherderPage(selenium, base_url).open()
+    assert page.unclassified_failure_count > 0
+
+    # Get next 50 jobs to see 'exception' unclassified failures
+    page.get_next_fifty_results()
+    page.deselect_all_failures()
+
+    # Test 'testfailed' unclassified failures
+    page.select_testfailed_failures()
+    page.open_next_unclassified_failure()
+    teststatus = page.job_details.job_result_status
+    assert teststatus in ['testfailed']
+    page.select_testfailed_failures()
+
+    # Test 'busted' unclassified failures
+    page.select_busted_failures()
+    page.open_next_unclassified_failure()
+    teststatus = page.job_details.job_result_status
+    assert teststatus in ['busted']
+    page.select_busted_failures()
```

Comment:
I think this should be a separate test. As you point out the test takes a long time to run, so splitting it into a test for each status will allow them to run in parallel. I do wonder if we need to test all of the status values though, as the functionality should be identical. If we do decide to cover multiple status values it could even make sense to make this a parameterised test to reduce duplication.